### DISCUSSION TITLE
A calculator for township could be very useful.

### DISCUSSION THREAD

**Discussion by gaberdo84** [AsherGlick/ResourceCalculator#109] [Ideas]
Lots of people would use it

**Comment by AsherGlick** [comment]
Ooh, the first "discussion" on resource calculator. If you could link whatever info you have about this game, for example things like:
* Storefronts for the game on whatever platforms it is on
* Already existing wikis or guide sites for the game which will make it easier to build data for the calculator
* Where the community of the game is, reddit, youtube, tiktok, etc
* Any other info you feel is useful

**Reply by ghost** [reply]
I think its a mobile game
